.github/workflows/lint_python.yml

```diff
@@ -0,0 +1,18 @@
+name: lint_python
+on:
+  pull_request:
+  push:
+  #  branches: [master]
+jobs:
+  lint_python:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - uses: actions/setup-python@master
+      - run: pip install black codespell flake8 isort pytest
+      - run: black . --diff || true
+      - run: codespell --quiet-level=2 || true  # --ignore-words-list="" --skip=""
+      - run: flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+      - run: isort --recursive . || true
+      - run: pip install -r requirements.txt || true
+      - run: pytest . || true
```

Comment:
Given that there are currently no tests, why run pytest?

Reply:
To be ready for the day when the project gets serious.

Reply:
There is a cost to having unused code in the project in terms of cognitive overhead, having to figure out what code deliberately is non-functional as opposed to being caused by a true bug, etc. so to be honest I'd rather no merge code that currently has no purpose and instead add the code at the time when it becomes required.